Insurgency/insserver

```diff
@@ -71,19 +114,19 @@ consolelogdate="${consolelogdir}/${servicename}-console-$(date '+%d-%m-%Y-%H-%M-
 # Do not edit
 
 fn_runfunction(){
+functiondir="${rootdir}/functions"
```

Comment:
I would like to merge your changes to fn_runfunction. Could you create a branch just for this change and I will merger this. :)